content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst

```diff
@@ -4,185 +4,229 @@ Process receipts and deliveries in two steps
+Process receipt
+---------------
 
-Click on the :guilabel:`Receipt` associated with the purchase order, then click :guilabel:`Validate`
-to complete the receipt and move the product to :guilabel:`WH/Input`.
+From the warehouse receipt form, the products ordered can be received into the warehouse. To receive
+the products, click :guilabel:`Validate`. Once validated, the receipt moves to the :guilabel:`Done`
+stage, and the products move to the :guilabel:`WH/Input` location.
 
 .. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-receipt-form.png
    :align: center
-   :alt: Receipt form for product move to WH/Input.
+   :alt: Receipt form for products ordered from vendor.
 
-Process the internal transfer
------------------------------
+Click back to the |PO| (via the breadcrumbs, at the top of the form) to view the |PO| form. On the
+product line, the quantity in the :guilabel:`Received` column now matches the ordered
+:guilabel:`Quantity`.
+
+Process internal transfer
+-------------------------
 
 Once the receipt is validated, an internal transfer is created and ready to process.
 
-Click back to the :guilabel:`Inventory Overview` in the breadcrumbs, and locate the
+To view the internal transfer, navigate to the :menuselection:`Inventory app`, and locate the
 :guilabel:`Internal Transfers` task card.
 
-Click the :guilabel:`# To Process` button to reveal a list of all internal transfers to process, and
-select the transfer associated with the previously validated receipt.
+Click the :guilabel:`# To Process` button on the task card to reveal a list of all internal
+transfers to process, and select the transfer associated with the previously validated receipt.
 
-Click :guilabel:`Validate` to complete the transfer and move the product from :guilabel:`WH/Input`
-to :guilabel:`WH/Stock`.
+Once ready, click :guilabel:`Validate` to complete the transfer and move the product from
+:guilabel:`WH/Input` to :guilabel:`WH/Stock`.
 
-Once the transfer is validated, the product enters inventory and is available for customer
+Once the transfer is validated, the products enter inventory, and are available for customer
 deliveries or manufacturing orders.
 
 .. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-internal-transfer.png
    :align: center
-   :alt: Validate the internal transfer to move the item to stock.
+   :alt: Internal transfer form for products ordered from vendor.
 
-Process a delivery order in two steps (pick + ship)
-===================================================
+Process delivery order in two steps (pick + ship)
+=================================================
 
-Create a sales order
---------------------
+When products are delivered in two steps, they move from warehouse stock to an output location, then
+from the output location to a customer location in the database upon validation of a picking order,
+and a subsequent delivery order (DO).
```

Comment:
```suggestion
When products are delivered in two steps, they move from warehouse stock to an output location validation of a picking order. Then
they move from the output location to a customer location upon validating the delivery order (DO).
```

I'm not sure if this is functionally correct, but the original sentence seemed a bit dense, trying to simplify it a bit more.

Reply:
I reworded it a bit from your suggestion but nice call - I broke the sentence up a bit to avoid running on. Suggestion implemented ✅